style-guide/template-style-guide.md

```diff
@@ -0,0 +1,141 @@
+that is         |  i.e.              |  Avoid non-English words
+
+## Topic types and templates
+
+This project recommends using the following templates from the
+[Good Docs project](https://github.com/thegooddocsproject/templates):
+
+- API Overview
+- API Quickstart
+- API Reference
+- Discussion Article
+- How To Guide
+- Logging Article
+- Reference Article
+- Tutorial
+
+## General writing tips
+
+{This section is optional.}
+
+For some general tips about improving writing see:
+
+- [Write the Docs - Style Guides](https://www.writethedocs.org/guide/writing/style-guides/#writing-style)
+- [18F Content Guide](https://content-guide.18f.gov/)
+
+## Accessible writing
+
+Documentation should be written in a way that supports people with disabilities
+and users with various input methods and devices. Improving accessibility also
+helps make documentation clearer and more useful for everyone.
+
+For resources on making your writing more accessible, see:
+
+- [Writing accessible documentation - Google developer documentation style guide](https://developers.google.com/style/accessibility)
+- [Accessibility guidelines and requirements - Microsoft Writing Style Guide](https://docs.microsoft.com/en-us/style-guide/accessibility/accessibility-guidelines-requirements)
+- [Writing for Accessibility - Mailchimp Content Style Guide](https://styleguide.mailchimp.com/writing-for-accessibility/)
+
+## Inclusive and bias-free writing
+
+When contributing to this project, you should strive to write documentation with
+inclusivity and diversity in mind. Inclusive language recognizes diversity and
+strives to communicate respectfully to all people. This kind of language is
+sensitive to differences and seeks to promote equal opportunities.
+
+For resources on making your writing more inclusive, see:
+
+- [Inclusive documentation - Google developer’s style guide](https://developers.google.com/style/inclusive-documentation)
+- [The Conscious Style Guide - a collection of resources](https://consciousstyleguide.com/)
+- [Bias-free communication - Microsoft Writing Style Guide](https://docs.microsoft.com/en-us/style-guide/bias-free-communication)
+- [Guidelines for Inclusive Language - Linguistic Society of America](https://www.linguisticsociety.org/resource/guidelines-inclusive-language)
+
+## Using linters
+
+{This section is optional.}
+
+This project uses the {your preferred linter.}
+
+{Provide instructions or policies related to the linter here.}
+
+## How the style guide is updated
+
+{Indicate here how frequently your style guide is reviewed, who owns the style
+guide, and how contributors can provide feedback on your style guide.}
+
+## Revision history
+
+{This section is optional or can be combined with the next section if needed.}
+
+The following table describes the history of all decisions and revisions made to
```

Comment:
```suggestion
This table describes the history of all decisions and revisions made to
```

Reply:
I'm not going to adopt this suggestion because I prefer my wording. It's also the convention throughout the document, so if I were to adopt this suggestion, I'd need to implement it in multiple places.

Reply:
NP.